wikimedia/wikipedia-ios
# [YiR] Present donation flow on top of YiR

**Phabricator:** https://phabricator.wikimedia.org/T376046

### Notes
These are changes based on Design and Product feedback. This PR presents the donation flow on top of Year in Review, instead of dismissing and pushing. This allows us to retain the user's place in the year in review flow. 

### Test Steps
1. Update device region to Mexico or Italy. Build up 6+ articles in History. Enable developer settings toggle. Background, foreground the app. YiR entry point should appear in Profile. 
2. Tap "Donate" on one of the YiR slides.
3. Donation form should now present on top of Year in Review. You should be able to close the donate form and return to your YiR slide.
